### DISCUSSION TITLE
Unretire a Kata

### DISCUSSION THREAD

**Discussion by seraph776** [codewars/codewars.com#2621] [Q&A] [answered]
I recently created a Kata. There was some issues with it, so I unpublished it. I corrected the issues, and now I am trying to re-publish the Kata but it will not allow me to. It says the Kata is `Retired` (screenshot below). 

![image](https://user-images.githubusercontent.com/72005563/156924130-5339dac8-4249-4e37-a391-755d5d153fdd.png)

**Question**: How do I publish this Kata?

**Comment by Blind4Basics** [comment, marked as answer]
Hi,

you cannot. Retired kata are in final state. You can still recreate it, but I'd advise you to consider the following:
* are you _sure_ (based on previous feeback) that the idea istelf is good/good enough?
* be sure you fixed every issue of this version before you publish the new version.

cheers
